src/MessagingBubble/MessagingBubble.tsx

```diff
@@ -32,44 +32,34 @@ export const MessagingBubble: React.FC<Props> = ({
   replyTo,
   attachments,
 }: Props) => {
+  const bubbleClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    theme === "ownMessage" ? cssClasses.MESSAGE_OWN : cssClasses.MESSAGE_OTHER,
+    replyTo && cssClasses.MESSAGE_REPLY_PARENT,
+  );
+
+  const replyClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    theme === "ownMessage" ? cssClasses.MESSAGE_REPLY_OWN : cssClasses.MESSAGE_REPLY_OTHER,
+  );
+
+  const attachmentClassNames = cx(
+    cssClasses.MESSAGE_BASE,
+    cssClasses.FLEXBOX,
+    theme === "ownMessage"
+      ? cssClasses.MESSAGE_ATTACHMENT_OWN
+      : cssClasses.MESSAGE_ATTACHMENT_OTHER,
+  );
```

Comment:
nice, very clean!

Reply:
Could also batch this into its own const, if you wanted to. 

```const themeClassName = theme === "ownMessage" ? cssClasses.MESSAGE_ATTACHMENT_OWN      : cssClasses.MESSAGE_ATTACHMENT_OTHER```

Maybe that is just unnecessary complication tho?